logproxy-client/src/main/java/com/oceanbase/clogproxy/client/LogProxyClient.java

```diff
@@ -50,6 +52,7 @@ public LogProxyClient(String host, int port, AbstractConnectionConfig config, Ss
         if (!config.valid()) {
             throw new IllegalArgumentException("Illegal argument for LogProxyClient");
         }
+        LogMessage.logTypeHelper = OBLogTypeHelper.OB_LOG_TYPE_HELPER;
```

Comment:
I think it's better to keep singleton in package level and use it externally by `LogTypeHelperFactory`.
Besides, we only use `OBLogTypeHelper` here, should we remove the other subclasses?

Reply:
Done